packages/react-test-renderer/src/ReactShallowRenderer.js

```diff
@@ -856,4 +856,8 @@ function getMaskedContext(contextTypes, unmaskedContext) {
   return context;
 }
 
+// This should probably be a default export and a named export.
```

Comment:
I don't understand what this comment is saying. Which one should it be?

Reply:
Both

Reply:
When we expose a proper ES module we can do:

```
export default ReactShallowRenderer;
export createRenderer;
```

It's just that if I do that, then Rollup doesn't do the magic thing to only use `module.exports = `